save-frontend/src/main/kotlin/org/cqfn/save/frontend/components/views/OwnerView.kt

```diff
@@ -0,0 +1,212 @@
+                            attrs.height = "260"
+                            attrs.width = "260"
+                        }
+                    }
+                }
+
+                div("position-relative") {
+                    attrs["style"] = kotlinext.js.jso<CSSProperties> {
+                        position = "relative".unsafeCast<Position>()
+                        textAlign = "center".unsafeCast<TextAlign>()
+                    }
+                    img(classes = "width-full color-bg-default") {
+                        attrs.src = "img/green_square.png"
+                        attrs.height = "200"
+                        attrs.width = "200"
+                    }
+                    div("position-absolute") {
+                        attrs["style"] = kotlinext.js.jso<CSSProperties> {
+                            top = "40%".unsafeCast<Top>()
+                            left = "40%".unsafeCast<Left>()
+                        }
+                        h1(" mb-0 text-gray-800") {
+                            +"4.5"
+                        }
+                    }
+                }
+            }
+
+            // ===================== RIGHT COLUMN =======================================================================
+            div("col-6") {
+                div("text-xs text-center font-weight-bold text-primary text-uppercase mb-3") {
+                    +"Projects"
+                }
+
+                child(tableComponent(
+                    columns = columns<Project> {
+                        column(id = "name", header = "Evaluated Tool", { name }) {
+                            buildElement {
+                                td {
+                                    a(href = "#/${it.row.original.owner}/${it.value}") { +it.value }
+                                    privacySpan(it.row.original)
+                                }
+                            }
+                        }
+                        column(id = "passed", header = "Description") {
+                            buildElement {
+                                td {
+                                    +(it.value.description ?: "Description not provided")
+                                }
+                            }
+                        }
+                        column(id = "rating", header = "Contest Rating") {
+                            buildElement {
+                                td {
+                                    +"0"
+                                }
+                            }
+                        }
+                    },
+                    initialPageSize = 10,
+                    useServerPaging = false,
+                    usePageSelection = false,
+                ) { _, _ ->
+                    get(
+                        url = "$apiUrl/projects/not-deleted",
```

Comment:
Where are you filtering projects by owner?

Reply:
Yes. Fixed